source/OutfitInfoDisplay.cpp

```diff
@@ -26,50 +26,52 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 using namespace std;
 
 namespace {
-	const map<string, double> SCALE = {
-		{"active cooling", 60.},
-		{"afterburner energy", 60.},
-		{"afterburner fuel", 60.},
-		{"afterburner heat", 60.},
-		{"cloak", 60.},
-		{"cloaking energy", 60.},
-		{"cloaking fuel", 60.},
-		{"cloaking heat", 60.},
-		{"cooling", 60.},
-		{"cooling energy", 60.},
-		{"energy consumption", 60.},
-		{"energy generation", 60.},
-		{"fuel consumption", 60.},
-		{"fuel energy", 60.},
-		{"fuel generation", 60.},
-		{"fuel heat", 60.},
-		{"heat generation", 60.},
-		{"heat dissipation", 60.},
-		{"hull repair rate", 60.},
-		{"hull energy", 60.},
-		{"hull fuel", 60.},
-		{"hull heat", 60.},
-		{"jump speed", 60.},
-		{"reverse thrusting energy", 60.},
-		{"reverse thrusting heat", 60.},
-		{"shield generation", 60.},
-		{"shield energy", 60.},
-		{"shield fuel", 60.},
-		{"shield heat", 60.},
-		{"solar collection", 60.},
-		{"thrusting energy", 60.},
-		{"thrusting heat", 60.},
-		{"turn", 60.},
-		{"turning energy", 60.},
-		{"turning heat", 60.},
+	const map<string, pair<double, string>> SCALE = {
+		{"active cooling", make_pair(60., "")},
```

Comment:
There's probably a better way of doing this that I don't know about instead of having a bunch of make_pairs.

Reply:
Since there will only be a few scale types, and many uses of those, you could define those pairs once in a vector, and then associate the attribute string to an index into that vector.

Reply:
Pictures, please.